lib/index.js

```diff
@@ -94,9 +94,9 @@ function from__1_2_0__to__2_0_0_rc1(asyncapi1, options) { // NOSONAR
       return out;
     });
   }
-  
+
   if (asyncapi1.topics) {
-    result.channels = _.mapKeys(result.topics, (_, topicName) => dotsToSlashes(`${asyncapi1.baseTopic ? `${asyncapi1.baseTopic}.` : ''}${topicName}`));
+    result.channels = _.mapKeys(result.topics, (__, topicName) => dotsToSlashes(`${asyncapi1.baseTopic ?? ''}${asyncapi1.baseTopic ? ".": ""}${topicName}`));
```

Comment:
Please move:

```
${asyncapi1.baseTopic ?? ''}${asyncapi1.baseTopic ? ".": ""} 
``` 

to the separate line and... simplify the logic :) It can be made even simpler.

Reply:
@magicmatatjahu Do you consider this simple enough?

``` javascript
result.channels = _.mapKeys(result.topics, (__, topicName) => {
      const baseTopic = asyncapi1.baseTopic ? `${asyncapi1.baseTopic}.` : ""
      return dotsToSlashes(`${baseTopic}${topicName}`)
    })
```

Reply:
By `separate line` I meant line above the `result.channels = _.mapKeys...` line, but I had in mind exactly this:

```js
const baseTopic = asyncapi1.baseTopic ? `${asyncapi1.baseTopic}.` : ""
```
 
 `asyncapi1` variable is variable declared before `_.mapKeys` logic and it is not needed to execute the logic for that in every iteration.

Reply:
Ahh. I understand now.